pkg/util/utils_test.go

```diff
@@ -23,3 +23,9 @@ import (
 func TestUtility(t *testing.T) {
 	// TODO: implement
 }
+
+func TestConstructPlanName(t *testing.T) {
```

Comment:
Since we never check for errors I have no idea what value this adds. Just kill it.

Reply:
ya, debug. making sure that it can't be misused.